src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Program.cs

```diff
@@ -89,6 +97,10 @@ public static async Task MainAsync(string[] args)
                         services.AddSingleton<ProjectSnapshotChangeTrigger, BackgroundDocumentGenerator>();
                         services.AddSingleton<RazorFileChangeDetectorManager>();
 
+                        // Options
+                        services.AddSingleton<RazorConfigurationService>();
+                        services.AddSingleton<IOptionsMonitor<RazorLSPOptions>, RazorLSPOptionsMonitor>();
```

Comment:
Would there ever be other options to monitor, or otherwise multiple implementations of `IOptionsMonitor`? I guess I'm trying to figure out why `IOptionsMonitor` gets an interface but `RazorConfigurationService` doesn't.

Reply:
Not entirely sure I understand the question. We won't need to monitor any other options. And `RazorConfigurationService` itself is a singleton and won't change and doesn't need to be monitored. We can talk offline if I didn't answer your question correctly.